google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/scrape/repository_services_api_helper.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+
+from . import constants
+
+
+class RepositoryServicesAPIHelper:
+
+    def __init__(self, server_address):
+        self.__base_api_endpoint = f'{server_address}/qrs'
+        self.__headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+            'X-Qlik-Xrfkey': constants.XRFKEY,
+            'User-Agent': 'Windows'
```

Comment:
Could you move the other header values to constants?

And also explain why the value Windows is used, since it is different than the usual `User-Agent` values.

Reply:
Moved the values to `constants.py` and added comments on the Windows header. Please let me know if it's clearer now :).